components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonInstallationDialogFragment.kt

```diff
@@ -206,6 +225,19 @@ class AddonInstallationDialogFragment(
         }
     }
 
+    override fun show(manager: FragmentManager, tag: String?) {
+        // This dialog is shown as a result of an async operation (installing
+        // an add-on). Once installation succeeds, the activity may already be
+        // in the process of being destroyed. Since the dialog doesn't have any
+        // state we need to keep, and since it's also fine to not display the dialog
+        // at all in case  the user navigates away, we can simply use
```

Comment:
Nit: double space `// at all in case  ..the user`

Reply:
👍 fixed.